weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterBar.tsx

```diff
@@ -453,6 +559,76 @@ export const FilterBar = ({
           </div>
         </Tailwind>
       </Popover>
+      <RemoveFilterModal
+        open={showDeleteWarning}
+        onClose={handleCancelDelete}
+        handleConfirmDelete={handleConfirmDelete}
+      />
```

Comment:
Thoughts on using the system dialog `confirm(` here to avoid adding new code?

Reply:
Martin and I collabed on using existing code